### PR TITLE
[HW-1] create login page

### PR DESCRIPTION
**Pull request by RomanPasternak19**
Screenshot from 2021-10-07 03-45-41

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
```
Comment (just0110) [inline package.json:1]: All configs, documentation, gitignore changes should not be in Login commit. Try next.

1. git reset --soft HEAD~1
2. via GUI interface of your IDEA chose configs, documentation, gitignore files. Create commit for it.
3. then chose all files related to Login and commit it
4. git push -f

File: src/mock/users.json [hunk 2]
```diff
@@ -0,0 +1,314 @@
+{
+    "results": [
```
Comment (just0110) [inline src/mock/users.json:2]: Login page?

File: src/modules/chat/types.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export interface Message {
```
Comment (just0110) [inline src/modules/chat/types.ts:1]: Login page?

File: src/modules/login/index.tsx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+                            <Text style={styles.text}>LOGIN</Text>
+                        </LinearGradient>
+                    </TouchableOpacity>
+                </View>
+            </View>
+        </View>
+    );
+}
+
+const styles = StyleSheet.create({
```
Comment (just0110) [inline src/modules/login/index.tsx:47]: To avoid SUPER CLASS, move common components and their logic and styles to separate files.

**Review by just0110 — CHANGES_REQUESTED** [review]
Make everything gradually. No need to hurry up

File: src/modules/login/index.tsx [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+import { Text, View } from '../../components/Themed';
+
+
+export default function LoginScreen() {
+
+    return (
+        <View style={styles.container}>
+            <LinearGradient colors={['#051937', '#004d7a', '#008793', '#00bf72', '#a8eb12']} style={styles.header}>
+                <View style={styles.logo}>
```
Comment (Andrey0295) [inline src/modules/login/index.tsx:15]: What about eslint?
With eslint you'll be able to do it more readable

**Review by Andrey0295 — COMMENTED** [review]
Your design looks cool, but your component is huge.
In Future, it'll be difficult to support)

**Pull request merged**